app/src/main/res/layout/activity_create_config.xml

```diff
@@ -32,11 +35,15 @@
                     android:layout_weight="@dimen/weight_one"
                     android:textSize="@dimen/text_size_of_label"
                     android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/create_config_margin1"
                     android:layout_gravity="center"
                     android:text="@string/select_instrument_title" />
 
                 <android.support.v7.widget.AppCompatSpinner
                     android:id="@+id/select_instrument_spinner"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"/>
                     android:layout_width="@dimen/width_zero"
                     android:layout_weight="@dimen/weight_two"
                     android:layout_height="wrap_content"
```

Comment:
Please remove the lines off your PR if they are not being used in the code anymore